readthedocs_ext/mixins.py

```diff
@@ -50,11 +50,14 @@ def copy_static_readthedocs_files(self):
                     context=ctx,
                 )
             else:
+                from sphinx.util.template import SphinxRenderer
```

Comment:
Why do we need to use this renderer? Could use a comment.

Reply:
This is just the new way. There's a comment about it here: https://github.com/readthedocs/readthedocs-sphinx-ext/blob/7b6c62b298d7b05a83550e089fc5672f5def5e44/readthedocs_ext/readthedocs.py#L302-L305